docker-compose.production.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.2"
+services:
+ db:
+   image: mysql:5.7
+   env_file:
+     - '.env'
+   ports:
+     - "3306:3306"
+   command: ['mysqld', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+   healthcheck:
+     test: ["CMD", "mysqladmin", "ping", "-h", "localhost","-plinks"]
+     timeout: 20s
+     interval: 10s
+     retries: 10
+ migrate:
+   image: wikipedialibrary/externallinks:latest
+   entrypoint: ["python", "/app/manage.py", "migrate"]
+   restart: on-failure
+   env_file:
+     - '.env'
+   depends_on:
+     - 'db'
+   volumes:
+     - ./:/app
+ externallinks_app:
+   image: wikipedialibrary/externallinks:latest
+   build: .
+   env_file:
+     - '.env'
+   depends_on:
+     - 'migrate'
+   ports:
+     - "8000:8000"
+   volumes:
+     - ./:/app
```

Comment:
This service should have the same name here and in the main file (where it's `app`). Since they have two different names, we'll have two separate services then the configs are merged at deploy time.
It should have a build target.
It only needs to be defined here if we want to keep the migration service in production and staging, but leave it off of the main docker-compose.